kong/tools/http.lua

```diff
@@ -210,16 +210,20 @@ _M.check_https = function(trusted_ip, allow_terminated)
   -- otherwise, we fall back to relying on the client scheme
   -- (which was either validated earlier, or we fall through this block)
   if trusted_ip then
-    local scheme = ngx.req.get_headers()["x-forwarded-proto"]
+    local scheme = ngx.var.http_x_forwarded_proto
+
```

Comment:
if you check value immediately below, then no newline here.

```suggestion
```

Reply:
done, thanks.